config.json

```diff
@@ -5,7 +5,13 @@
   "active": false,
   "test_pattern": "TODO",
   "exercises": [
-
+    {
+      "slug": "leap",
+      "difficulty": 1,
+      "topics": [
+        "generators"
```

Comment:
The example solution doesn't have any generators, perhaps change this to `control-flow (if-else statements)` or `comparisons` or similar?